pkgs/by-name/se/seabios/package.nix

```diff
@@ -29,20 +33,28 @@ stdenv.mkDerivation (finalAttrs: {
     "EXTRAVERSION=\"-nixpkgs\""
   ];
 
-  hardeningDisable = [ "pic" "stackprotector" "fortify" ];
+  hardeningDisable = [
+    "pic"
+    "stackprotector"
+    "fortify"
+  ];
 
-  postConfigure = let
-    config = writeText "config.txt" (lib.generators.toKeyValue { } {
-      # SeaBIOS with CSM (Compatible Support Module) support; learn more at
-      # https://www.electronicshub.org/what-is-csm-bios/
-      "CONFIG_CSM" = "y";
-      "CONFIG_PERMIT_UNALIGNED_PCIROM" = "y";
-      "CONFIG_QEMU_HARDWARE" = "y";
-    });
-  in ''
-    cp ${config} .config
-    make olddefconfig
-  '';
+  postConfigure =
+    let
+      config = writeText "config.txt" (
```

Comment:
What about accepting config as argument? Then it can be reused for qubes seabios builders

Reply:
Actually, nevermind, makes little sense to reuse that.

Reply:
That's just formatting. Anderson used a generator to write the necessary build configuration.